napari/_qt/qt_hidpi_settings.py

```diff
@@ -0,0 +1,34 @@
+"""
+Check screen resolution to ensure text rendering works for all screen resolutions.
+"""
+
+import sys
+from qtpy.QtWidgets import QApplication
+from qtpy.QtCore import QCoreApplication, Qt
+
+
+def dpis():
+    """Determine resolution of all connected monitors
+
+    see reference,
+    https://stackoverflow.com/questions/54271887/calculate-screen-dpi
+
+    Returns:
+        list -- Containing the DPI values of connected monitors
+    """
+    app = QApplication(sys.argv)
+    dpi_values = [screen.physicalDotsPerInch() for screen in app.screens()]
+    app.quit()
+    return dpi_values
+
+
+def adjust_dpi_settings(dpi_threshold=150):
+    """Set HighDpiScaling to True if any screen DPI is greater than threshold.
+
+    Keyword Arguments:
+        dpi_threshold {int} -- Threshold for HighDpiScaling to be switched on (default: {150})
+    """
+
+    currentdpis = dpis()
+    if any(dpi > dpi_threshold for dpi in currentdpis):
+        QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling, True)
```

Comment:
why not always set this?

Reply:
@hmaarrfk I incorrectly thought that turning this on would make things look huge on low DPI screens! Actually it just works! :tada: thanks for the suggestion! Why isn't this just on by default! :man_shrugging: